packages/react-dom/src/__tests__/ReactDOMEventPropagation-test.js

```diff
@@ -759,7 +810,7 @@ describe('ReactDOMEventListener', () => {
           node.dispatchEvent(
             new Event('transitionend', {
               bubbles: true,
-              cancelable: true,
+              cancelable: false,
```

Comment:
Doesn't really matter since synthetic events don't need to match real events. But better to test them closer to what they'll look when dispatched from user agents.